internal/docs/ESRPCodeSign.md

```diff
@@ -10,4 +10,4 @@ I have followed this [doc](https://eng.ms/docs/more/containers-secure-supply-cha
 For verification of signing we can do through 2 ways.
 
 1. Locally through the doc https://eng.ms/docs/more/containers-secure-supply-chain/signing under validation section using notation. We have to use our own [certificate](https://ms.portal.azure.com/#view/Microsoft_Azure_KeyVault/ListObjectVersionsRBACBlade/~/overview/objectType/certificates/objectId/https%3A%2F%2Fesrpprometheuskv.vault.azure.net%2Fcertificates%2FESRPReqPrometheusCert/vaultResourceUri/%2Fsubscriptions%2F9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb%2FresourceGroups%2FESRPPrometheus%2Fproviders%2FMicrosoft.KeyVault%2Fvaults%2FESRPPrometheusKV/vaultId/%2Fsubscriptions%2F9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb%2FresourceGroups%2FESRPPrometheus%2Fproviders%2FMicrosoft.KeyVault%2Fvaults%2FESRPPrometheusKV) instead of the one in the example.
-2. We can do a docker pull on the signed images and it will not have the following error message - "manifest verification failed for digest sha256..."
+2. We can do a "docker manifest inspect -v <image>" on the signed images and it will not have the following error message - "manifest verification failed for digest sha256..."
```

Comment:
Soham, from what we investigated looks like it only works with specific versions of docker isn't it? Do we need to check what works reliably with the latest version of docker?

Reply:
Yes Rashmi. That's why I have added a task https://msazure.visualstudio.com/InfrastructureInsights/_sprints/taskboard/Prometheus-agent/InfrastructureInsights/Germanium/CY23Q4/Monthly/10%20Oct%20(Oct%2001%20-%20Oct%2031)?workitem=25343201 to implement a verification step in image signing in the ADO yaml script so that it only pulls signed images.

Reply:
Okay, thanks Soham